btc-transaction-helper.js

```diff
@@ -245,6 +244,10 @@ class BtcTransactionHelper{
         return await this.nodeClient.mine(blocks);
```

Comment:
`btcClient.generate` can mine more than 1 block when called. It will be called like `btcClient.generate(1)`, or `btcClient.generate(3)` and it will mine that amount. This `this.nodeClient.mine` calls that `generate` function.